doc/classes/AudioStreamPlayer.xml

```diff
@@ -127,6 +129,7 @@
 			Bus on which this audio is playing.
 		</member>
 		<member name="mix_target" type="int" setter="set_mix_target" getter="get_mix_target" enum="AudioStreamPlayer.MixTarget">
+			In case you have an audio setup with more than stereo speakers, this defines, to which speaker-channels the [member stream] is played.
```

Comment:
Too many commas.

If the audio configuration has more than two speakers, this sets the target channels. See [code]MIX_TARGET_*[/code] constants.